moninag/static/src/ts/app/checks/check-list.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { Check } from './check';
+import { CheckAddComponent } from './check-add.component';
+import { ChecksService } from './checks.service';
+import { Observable } from 'rxjs/Observable';
+import { Plugin } from './plugin';
+import { Service } from '../services/services';
+
+
+@Component({ 
+    selector:'checks-list',
+    template:`
+    <div class="table-responsive">
+        <ul style="list-style-type:none; padding:0">          
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                        <th>Plugin name</th>
+                        <th>Status</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><li *ngFor="let check of service.checks" (click)="onSelect(check); gotoDetail()" value={{check.id}>{{check.name}}</li></td>
+                        <td><li *ngFor="let check of service.checks" (click)="onSelect(check); gotoDetail()" value={{check.id}>{{check.plugin_name}}</li></td>
+                        <td><li *ngFor="let check of service.checks" (click)="onSelect(check); gotoDetail()" value={{check.id}>{{check.status}}</li></td>
+                    </tr>
+                </tbody>
+            </table>
+        </ul>
+        <button (click)="onSelect2(service)"> Add new check </button>
+        <div *ngIf="selectedService">
+            <checkadd-app></checkadd-app>
+        </div>
+    </div>
+    `,
+    providers: [ChecksService],
+    styles: [`
+       table, th, td {
```

Comment:
The next task is to replace styles with angular material, so I don't change it here.